src/ui/buttons/button-animations/fadeout-logo-show-label-text.jsx

```diff
@@ -0,0 +1,144 @@
+/* @flow */
+/** @jsx node */
+import { LOGO_CLASS } from '@paypal/sdk-logos/src';
+import { node, Fragment, type ChildType } from 'jsx-pragmatic/src';
+
+import { CLASS } from '../../../constants';
+import { BUTTON_SIZE_STYLE } from '../config';
+
+import type { ButtonAnimationOutputParams, LabelOptions, DivideLogoAnimationProps } from './types';
+
+export const ANIMATION = {
+    LABEL_CONTAINER: ('fadeout-logo-show-label-animation-container' : 'fadeout-logo-show-label-animation-container'),
+    CONTAINER:       ('fadeout-logo-show-label-animation' : 'fadeout-logo-show-label-animation')
+};
+
+export function AnimationComponent({ animationLabelText } : LabelOptions) : ChildType {
+    // experimentName must match elmo experiment name
+    const config = {
+        labelText:      animationLabelText,
+        labelClass:     ANIMATION.LABEL_CONTAINER,
+        experimentName: 'Varied_Button_Design'
+    };
+   
+    return (
+        <Fragment>
+            <div class={ config.labelClass } data-experiment={ config.experimentName }> <span>{config.labelText}</span></div>
+            <style innerHTML={ `
+                .${ CLASS.DOM_READY } .${ ANIMATION.CONTAINER } img.${ LOGO_CLASS.LOGO }{
+                    position: relative;
+                }
+                
+                .${ ANIMATION.CONTAINER } .${ ANIMATION.LABEL_CONTAINER } {
+                    position: fixed;
+                    opacity: 0; 
+                    color: #142C8E;
+                    font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+                    font-size: 10px;
+                }
+            ` } />;
+        </Fragment>
+    );
+}
+
+// Returns label container if the button sizes match
+const getAnimationProps = function(document, configuration) : DivideLogoAnimationProps | null {
```

Comment:
Maybe rename the type to `LogoAnimationProps` ?

Reply:
no this properties are not just for logo, so getAnimationProps is more generic

Reply:
@erick2014 I said the type not the function name. AnimationProps is fine too

Reply:
@erick2014 Do NOT mark my review comments as resolved without asking me first.

Reply:
Type was renamed, please check